web/web.go

```diff
@@ -86,6 +86,23 @@ func IsWebhookCall(a *app.App, r *http.Request) bool {
 	return strings.HasPrefix(r.URL.Path, path.Join(subpath, "hooks")+"/")
 }
 
+func IsOAuthApiCall(config configservice.ConfigService, r *http.Request) bool {
+	subpath, _ := utils.GetSubpathFromConfig(config.Config())
+
+	// To avoid the /oauth/authorize page on get
+	if r.Method == "GET" {
```

Comment:
0/5 but would it better to check for that endpoint path specifically versus adding an exception for all GET requests?

Reply:
Done! PTAL